subprojects/packagefiles/fdk-aac/meson.build

```diff
@@ -391,14 +391,16 @@ if get_option('remove-date')
 endif
 
 fdk_aac = library(
-  'fdk_aac',
+  'fdk-aac',
   fdk_aac_src,
   dependencies: m_dep,
   include_directories: fdk_aac_inc,
   install: true,
   cpp_args: cc.get_supported_arguments('-fno-exceptions', '-fno-rtti') + extra_args,
   c_args: extra_args,
   vs_module_defs: 'fdk-aac.def',
+  soversion: '2',
```

Comment:
The value of `version` will cause `soversion` to default to `2`, so this is redundant.